### DISCUSSION TITLE
How to rename a clan

### DISCUSSION THREAD

**Discussion by transcental** [RoinujNosde/SimpleClans#427] [Q&A] [answered]
Hi, I see from #360 tat it's possible to rename a clan, but this isn't in the docs and I've been unable to find it ingame. Would it be possible to add it, or for someone to tell me how to do it please?

**Comment by transcental** [comment]
Ok, I've figured it out but I'd appreciate if it could be added to the docs and if players with level 4 op don't need the permission too.

**Reply by Tomut0** [reply, marked as answer]
Thank you for noticing. I had added it to docs.
`/clan mod rename` command can be accessed by `simpleclans.mod.rename` permission node.
